Wikipedia/Code/NotificationsCenterCellStyle.swift

```diff
@@ -0,0 +1,176 @@
+	}
+
+	func subheaderFont(_ displayState: NotificationsCenterCellDisplayState) -> UIFont {
+		if displayState.isRead {			
+			return UIFont.wmf_font(.subheadline, compatibleWithTraitCollection: traitCollection)
+		}
+
+		return UIFont.wmf_font(.semiboldSubheadline, compatibleWithTraitCollection: traitCollection)
+	}
+
+	var messageFont: UIFont {
+		return UIFont.wmf_font(.body, compatibleWithTraitCollection: traitCollection)
+	}
+
+	func metadataFont(_ displayState: NotificationsCenterCellDisplayState) -> UIFont {
+		if displayState.isRead {
+			return UIFont.wmf_font(.footnote, compatibleWithTraitCollection: traitCollection)
+		}
+
+		return UIFont.wmf_font(.mediumFootnote, compatibleWithTraitCollection: traitCollection)
+	}
+
+	func relativeTimeAgoFont(_ displayState: NotificationsCenterCellDisplayState) -> UIFont {
+		if displayState.isRead {
+			return UIFont.wmf_font(.footnote, compatibleWithTraitCollection: traitCollection)
+		}
+
+		return UIFont.wmf_font(.boldFootnote, compatibleWithTraitCollection: traitCollection)
+	}
+
+	var projectSourceFont: UIFont {
+		return UIFont.wmf_font(.caption2, compatibleWithTraitCollection: traitCollection)
+	}
+
+	// MARK: - Images
+
+	func leadingImage(_ displayState: NotificationsCenterCellDisplayState) -> UIImage? {
+		guard !displayState.isEditing else {
+            let symbolConfiguration = UIImage.SymbolConfiguration(weight: .bold)
+            let image = displayState.isSelected ? UIImage(systemName: "checkmark", withConfiguration: symbolConfiguration) : nil
+            return image
+		}
+
+        // Return image for the notification type
+        switch notificationType {
+        case .userTalkPageMessage:
+            return UIImage(named: "notifications-type-user-talk-message")
+        case .mentionInTalkPage, .mentionInEditSummary, .successfulMention:
+            return UIImage(systemName: "at", withConfiguration: UIImage.SymbolConfiguration(weight: .heavy))
+        case .editReverted:
+            return UIImage(named: "notifications-type-edit-revert")
+        case .userRightsChange:
+            return UIImage(named: "notifications-type-user-rights")
+        case .pageReviewed:
+            return UIImage(named: "notifications-type-page-reviewed")
+        case .pageLinked, .connectionWithWikidata:
+            return UIImage(systemName: "link", withConfiguration: UIImage.SymbolConfiguration(weight: .semibold))
+        case .thanks:
+            return UIImage(named: "notifications-type-thanks")
+        case .welcome, .translationMilestone(_), .editMilestone:
+            return UIImage(systemName: "heart.fill")
+        case .loginFailKnownDevice, .loginFailUnknownDevice, .loginSuccessUnknownDevice,
+             .unknownSystemAlert, .unknownAlert:
+            return UIImage(named: "notifications-type-login-notify")
+        case .emailFromOtherUser:
+            return UIImage(systemName: "mail", withConfiguration: UIImage.SymbolConfiguration(weight: .semibold))
```

Comment:
FYI, discovered this after initial review - it doesn't look like this system icon shows up in iOS13.

Reply:
Great catch on this – looks like the `mail` SF Symbol is iOS 14+. I replaced it with `envelope` for iOS 13. I could have just done some nil coalescing (`??`) [here](https://github.com/wikimedia/wikipedia-ios/pull/4041/commits/c28b97ea70e4d76ecd3c073d5fc526f809c3b83f#diff-b450f3bce014695328d8d0c58c2a112de015165cfe0912a26fd36369741c97f2) to be more direct, but I think calling out the `if #available` is valuable here to communicate in code the reason for this fallback. I also audited the rest of these notification type images and I believe that was the only case of iOS 14+ specific SF Symbols here.

We do use `doc.plaintext.fill` which is iOS 14+ in the `NotificationCenterCell`’s `metaActionButton`, but that is temporary – I added a comment.